dashboard/app/models/script.rb

```diff
@@ -34,7 +34,7 @@ class Script < ActiveRecord::Base
 
   include SerializedProperties
 
-  serialized_attrs %w(pd)
+  serialized_attrs %w(pd admin_required)
```

Comment:
login_required is a column in the db (see the db schema comments above), which I think was added before serialized_attrs existed. I added admin_required to serialized_attrs (which means its in a hash that gets serialized to the properties text column), which doesn't require a migration.